CRITICAL - 2023-12-02 04:42:27 --> Invalid file: "products.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(11): view('products')
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
CRITICAL - 2023-12-02 04:42:28 --> Invalid file: "products.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(11): view('products')
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
CRITICAL - 2023-12-02 04:42:46 --> Invalid file: "products.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(11): view('products')
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:07:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#10 {main}
CRITICAL - 2023-12-02 06:07:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:10:41 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#10 {main}
CRITICAL - 2023-12-02 06:10:41 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:11:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#10 {main}
CRITICAL - 2023-12-02 06:11:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:11:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#10 {main}
CRITICAL - 2023-12-02 06:11:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:11:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#10 {main}
CRITICAL - 2023-12-02 06:11:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:11:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', Array)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#10 {main}
CRITICAL - 2023-12-02 06:11:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(871): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\ProductController.php(21): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:11:58 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\BaseModel->findAll()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\BaseModel->findAll()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}
CRITICAL - 2023-12-02 06:11:58 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProductController.php(21): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
ERROR - 2023-12-02 06:12:34 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', 'ci_project1', 3306, '', 0)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\BaseModel->findAll()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\app\Controllers\ProductController.php(21): CodeIgniter\BaseModel->findAll()
#6 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(942): App\Controllers\ProductController->index()
#7 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 F:\Mizanur_Rahman_1277622\XAMPP_7.4.3\htdocs\codeigniter\ci_project1\system\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahm...')
#11 {main}
CRITICAL - 2023-12-02 06:12:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProductController.php(21): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\ProductController->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('F:\\Mizanur_Rahman_1277622\\XAMPP_7.4.3\\htdocs\\codeigniter\\ci_project1\\public\\index.php')
